### ISSUE TITLE
Getting the unrolled topology of an onxx model

### ISSUE THREAD

**Issue by nishraptor** [onnx/onnx#3659]
Is there a way to get the adjacency list that describes an entire model in terms of connectivity? I'd like to get this adjacency list including unrolled convolutions, but I haven't found any way to do this. Is this possible?

**Comment by jcwchen** [comment]
Hi @nishraptor,
Yes, it's possible. You can traverse `onnx_model.graph.node` to get the connection relationship (input to output) and maintain a dictionary/map for an adjacency list. Also please note that a valid ONNX model will have a topologically sorted graph.node. Thanks.

**Comment by nishraptor** [comment]
Thanks,

I downloaded alexnet and was trying to iterate through this. I got the following:

`input: "data_0"
input: "conv1_w_0"
input: "conv1_b_0"
output: "conv1_1"
name: ""
op_type: "Conv"
attribute {
  name: "strides"
  ints: 4
  ints: 4
  type: INTS
}
attribute {
  name: "pads"
  ints: 0
  ints: 0
  ints: 0
  ints: 0
  type: INTS
}
attribute {
  name: "kernel_shape"
  ints: 11
  ints: 11
  type: INTS
.
.
.
}
`
How would I go about getting the map? Would I have to manually keep track of how many neurons are in each layer and connect them with my own logic?
